docs/related.md

```diff
@@ -28,6 +28,8 @@ dispatches to 20+ backends. Some differences with Narwhals are:
   from non-pandas environments, the difference in the relative size increase is ~1000x
 - Narwhals only supports 4 backends, Ibis more than 20
 - Narwhals is focused on fundamental dataframe operations, Ibis on SQL backends
+- Narwhals has negligible overhead for dataframe backends, while
+  [Ibis' overhead is significant](https://github.com/ibis-project/ibis/issues/9345)
```

Comment:
Maybe 
```suggestion
  [Ibis' overhead can be significant](https://github.com/ibis-project/ibis/issues/9345)
```
😇

Not sure if that's the case for all dataframe libraries

Reply:
it's about 2x for Polars, 4x for pandas

thanks, addressed in b41b267

I'm honestly not sure whether to publish a comparison or whether that'd be perceive as "too mean"

Reply:
update: the Polars overhead isn't consistent, sometimes it's on-par, sometimes it's slower, sometimes slightly faster

i haven't checked, but I presume it's because they leverage Polars' SQL engine, whereas for pandas they do a direct translation?

Regarding their pandas backend, they don't recommend using it at all, and say it's mostly there for historical reasons. so I don't think a public comparison is warranted. But I think this is useful to answer the question "why not just use Ibis"